src/components/Dashboard/Views/Table/TableView.vue

```diff
@@ -46,19 +54,30 @@ const getTorrentRowColorClass = (torrent: TorrentType) => [isTorrentSelected(tor
 </script>
 
 <template>
-  <v-table id="torrentList" class="pa-0" density="compact">
-    <thead>
+  <v-data-table
+    id="torrentList"
+    density="compact"
+    :mobile="false"
+    :headers="headers"
+    :items="paginatedTorrents"
+    :height="height"
+    :item-height="36"
+    items-per-page="-1"
+    hide-default-footer
+    fixed-header>
+    <template #headers="{ columns }">
       <tr>
-        <th class="px-1" />
-        <th v-if="dashboardStore.isSelectionMultiple" />
-        <Header :title="$t('torrent.properties.name')" sort-key="name" @onHeaderClick="onHeaderClick('name')" />
-        <Header v-for="ppt in torrentProperties" :title="$t(ppt.props.titleKey)" :sort-key="ppt.sortKey" @onHeaderClick="onHeaderClick(ppt.sortKey)" />
+        <template v-for="header in columns" :key="header.key">
+          <th v-if="header.key === 'statusIndicator'" class="px-1" />
+          <th v-else-if="header.key === 'multipleSelectionCheckbox'" />
+          <Header v-else :title="header.title!" :sortKey="header.key!" @onHeaderClick="onHeaderClick(header.key as keyof Torrent)" />
+        </template>
       </tr>
-    </thead>
-    <tbody>
+    </template>
+
+    <template #item="{ item: torrent }">
       <tr
-        v-for="torrent in paginatedTorrents"
-        :class="['cursor-pointer', 'selected', 'ripple-fix', getTorrentRowColorClass(torrent)]"
+        :class="['cursor-pointer', 'selected', getTorrentRowColorClass(torrent)]"
```

Comment:
This reverts to ripple effect issue found with Safari browser. Any reason for ommiting `ripple-fix`?

Reply:
I checked the code and bo class with that name exists in our CSS. I don't remember having fixed that issue...

Reply:
#1864. It's still a valid workaround.

Reply:
Well I may have messed something up while looking for it, thanks for the review!

Reply:
Thank you too!